app/cdap/components/WrangleHome/Components/OngoingDataExplorationsCard/styles.ts

```diff
@@ -19,7 +19,20 @@ import { green } from '@material-ui/core/colors';
 import { red } from '@material-ui/core/colors';
 
 export const useStyles = makeStyles({
-  gridContainer: {
+  gridContainerWorkspaces: {
+    width: 1306,
+    height: 77,
+    border: '1px solid #DADCE0',
```

Comment:
mui colors?

Reply:
Color not available in mui color palette